software/prisme-analytics.yml

```
@@ -1,15 +1,14 @@
 name: Prisme Analytics
-website_url: https://www.prismeanalytics.com
+website_url: https://github.com/prismelabs/analytics
```

```suggestion
website_url: https://www.prismeanalytics.com
```

website is back online